python/lsst/ctrl/bps/wms_service.py

```diff
@@ -173,6 +173,16 @@ class BaseWmsService:
     def __init__(self, config):
         self.config = config
```

Comment:
At the moment I think it might be better to let the WMS plugin to override this method (with the base class having no actual defaults we won't be able to write unit test(s) to verify if it works as intended).